# Fix-up incorrect syntax in makefiles
#
diff --git a/10.03_app_demo/2_src/makefile b/10.03_app_demo/2_src/makefile
index 1a8ad86..f494b78 100644
--- a/10.03_app_demo/2_src/makefile
+++ b/10.03_app_demo/2_src/makefile
@@ -1,9 +1,9 @@
 # Main switch between UNIBUS and QBUS makefile
 
 ifeq ($(QUNIBONE_PLATFORM),UNIBUS)
-        include makefile_u
+include makefile_u
 else ifeq  ($(QUNIBONE_PLATFORM),QBUS)
-        include makefile_q
+include makefile_q
 else
-	$(error Set QUNIBONE_PLATFORM to UNIBUS or QBUS!)
+$(error Set QUNIBONE_PLATFORM to UNIBUS or QBUS!)
 endif
diff --git a/10.03_app_demo/2_src/makefile_q b/10.03_app_demo/2_src/makefile_q
index 4cacc34..d3dc0c5 100644
--- a/10.03_app_demo/2_src/makefile_q
+++ b/10.03_app_demo/2_src/makefile_q
@@ -32,22 +32,22 @@ LDFLAGS+= -static -lstdc++ -lpthread $(PRUSS_DRV_LIB)
 
 # compiler flags and libraries
 ifeq ($(MAKE_CONFIGURATION),RELEASE)
-	CC_DBG_FLAGS = -O3 -Wall -Wextra -Wshadow -DDBG
+CC_DBG_FLAGS = -O3 -Wall -Wextra -Wshadow -DDBG
 else ifeq ($(MAKE_CONFIGURATION),DBG)
- 	CC_DBG_FLAGS = -ggdb3 -O0 -Wall -Wextra -Wshadow -DDBG
+CC_DBG_FLAGS = -ggdb3 -O0 -Wall -Wextra -Wshadow -DDBG
 else
-	$(error Set MAKE_CONFIGURATION to RELEASE or DBG!)
+$(error Set MAKE_CONFIGURATION to RELEASE or DBG!)
 endif
 
 ifeq ($(MAKE_TARGET_ARCH),BBB)
-	# cross compile on x64 for BBB
-	CC=$(BBB_CC)
-	OS_CCDEFS = -DARM -U__STRICT_ANSI__
-	OBJDIR=$(abspath ../4_deploy_q)
+# cross compile on x64 for BBB
+CC=$(BBB_CC)
+OS_CCDEFS = -DARM -U__STRICT_ANSI__
+OBJDIR=$(abspath ../4_deploy_q)
 else
-	# local compile on BBB
-	OS_CCDEFS = -DARM -U__STRICT_ANSI__
-	OBJDIR=$(abspath ../4_deploy_q)
+# local compile on BBB
+OS_CCDEFS = -DARM -U__STRICT_ANSI__
+OBJDIR=$(abspath ../4_deploy_q)
 endif
 
 
diff --git a/10.03_app_demo/2_src/makefile_u b/10.03_app_demo/2_src/makefile_u
index 2aa1498..f85462b 100644
--- a/10.03_app_demo/2_src/makefile_u
+++ b/10.03_app_demo/2_src/makefile_u
@@ -32,22 +32,22 @@ LDFLAGS+= -static -lstdc++ -lpthread $(PRUSS_DRV_LIB)
 
 # compiler flags and libraries
 ifeq ($(MAKE_CONFIGURATION),RELEASE)
-	CC_DBG_FLAGS = -O3 -Wall -Wextra -Wshadow -DDBG
+CC_DBG_FLAGS = -O3 -Wall -Wextra -Wshadow -DDBG
 else ifeq ($(MAKE_CONFIGURATION),DBG)
- 	CC_DBG_FLAGS = -ggdb3 -O0 -Wall -Wextra -Wshadow -DDBG
+CC_DBG_FLAGS = -ggdb3 -O0 -Wall -Wextra -Wshadow -DDBG
 else
-	$(error Set MAKE_CONFIGURATION to RELEASE or DBG!)
+$(error Set MAKE_CONFIGURATION to RELEASE or DBG!)
 endif
 
 ifeq ($(MAKE_TARGET_ARCH),BBB)
-	# cross compile on x64 for BBB
-	CC=$(BBB_CC)
-	OS_CCDEFS = -DARM -U__STRICT_ANSI__
-	OBJDIR=$(abspath ../4_deploy_u)
+# cross compile on x64 for BBB
+CC=$(BBB_CC)
+OS_CCDEFS = -DARM -U__STRICT_ANSI__
+OBJDIR=$(abspath ../4_deploy_u)
 else
-	# local compile on BBB
-	OS_CCDEFS = -DARM -U__STRICT_ANSI__
-	OBJDIR=$(abspath ../4_deploy_u)
+# local compile on BBB
+OS_CCDEFS = -DARM -U__STRICT_ANSI__
+OBJDIR=$(abspath ../4_deploy_u)
 endif
 
 
